Add Chapter 3.C rules to "all the rules we know"

llewelld opened the pull request:
The usual rules dump; for 3C.

Preview of the changes:
https://github.com/llewelld/hut23-linear-algebra/blob/gh-action-result/pdf-files/reference/all-the-rules-we-know.pdf

triangle-man commented on reference/all-the-rules-we-know.tex:
```diff
@@ -746,4 +747,220 @@ \section*{Chapter 3.B}
+\end{result}
+
+\begin{result}{3.40}[$\dim \F^{m, n} = m n$]
+Suppose $m$ and $n$ are positive integers. With addition and scalar multiplication defined as above, $\F^{m, n}$ is avector space of dimention $m n$.
```
Typo: "dimention" should be "dimension"

llewelld replied:
Likewise, thanks for flagging this. It's fixed now.

triangle-man reviewed (commented):
Looks good to me! I added comments where I spotted typos.

PS. for kernel and range the "modern" approach is `\DeclareMathOperator{\kernel}{null}`

llewelld commented on reference/all-the-rules-we-know.tex:
```diff
@@ -746,4 +747,220 @@ \section*{Chapter 3.B}
+\section*{Chapter 3.C}
+
+\begin{definition}{3.29}[matrix, $A_{j, k}$]
+Suppose $m$ and £n£ are nonnegative integers. An $m$-by-$n$ \defn{matrix} $A$ is a rectangular array of elements of $\F$ with $m$ rows and $n$ columns:
```
Thanks, nicely spotted; now fixed.

llewelld commented on reference/all-the-rules-we-know.tex:
```diff
@@ -746,4 +747,220 @@ \section*{Chapter 3.B}
+\end{result}
+
+\begin{result}{3.40}[$\dim \F^{m, n} = m n$]
+Suppose $m$ and $n$ are positive integers. With addition and scalar multiplication defined as above, $\F^{m, n}$ is avector space of dimention $m n$.
```
And again; thank you!

llewelld commented:
Thanks for the review @triangle-man. I've fixed the typos and also updated the operator definitions. `\DeclareMathOperator` was new to me, so thanks for making me aware.

Also squashed and rebased.

llewelld commented:
Rebased.

llewelld commented:
Thanks @rwood-97!

llewelld commented:
Rebased.
